hwdef: fix luminousbee5 parameters

robertlong13 opened the pull request:
This fixes an errant rename of GPS_SBAS_TYPE to GPS1_SBAS_TYPE that occurred in https://github.com/ArduPilot/ardupilot/pull/26497.

I also removed the skybrush-fork parameters and added a blank include file for them to populate on their fork. This keeps our new CI checks (that I used to catch the above GPS mistake) from complaining without needing to keep a list of non-existent param patters to ignore. As a maintainer of some forks myself, I think this is an easier pattern. FYI @ntamas

ntamas reviewed (approved):
Looks good to me! One question: now that you fixed the param file, are you sure you want to remove `luminousbee5` from `param_check_all.py`? Wasn't the whole point of this PR to make the file pass the param checks?

robertlong13 commented:
> Looks good to me! One question: now that you fixed the param file, are you sure you want to remove `luminousbee5` from `param_check_all.py`?

I've removed it from the list of boards to _skip_. When I added the check, I just dumped every failing board into that list so I could come back to them as separate PRs.

ntamas commented:
Ah okay then - sorry, it's 3am here so I kinda missed the point.
